scan_joints_arm: 
    - [0.010523236721068734, -1.4790833239639014, 0.10027132190633238, -2.416246868493765, 0.08994288870361117, 1.4022499498261343, 0.8516519819506339]
    - [-0.5204305512655648, -0.7962560020246003, 0.9121961832112832, -1.6720657872605567, 0.0914094145960278, 1.2810500415696036, 0.8710614908889162]
    - [0.03202341903301707, 0.45900370514601985, 0.0743635250858064, -0.8394780465249851, 0.01546591704007652, 0.7776030993991428, 0.8335337665490805]
    - [0.0927063864391347, -0.6268411712966925, -0.5511789947129442, -1.8378490985067266, 0.1656381993108548, 1.2387742138438755, 0.9008788375077649]
drop_joints_arm: [1.8125419079753868, 0.35502879822802413, -0.044053724205053386, -2.5928146389998137, 0.0011553696715772578, 2.9127716170698204, 0.94913]
home_joints_arm: [0.0, -0.785, 0.0, -2.356, 0.0, 1.57, 0.785]
ready_joints_arm:  [0.0, -0.785, 0.0, -2.356, 0.0, 1.57, 0.785]
search_joints_arm: [0.7245, -1.4537, -0.1218, -2.7534146704189664, -0.14563071170575342, 2.547758875370693, 0.7835325231057125]

base_frame_id: "panda_link0"

arm_velocity_scaling_drop: 0.3
arm_velocity_scaling_grasp: 0.2

table_height: 0.01   # Measured from the base frame specified above

grasp_selection_method: "auto"    # choice: manual, auto
pregrasp_offset_z: -0.05
grasp_offset_z: 0.05  # grasp frame is defined at the gripper's palm
